<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-2G86F66K" ignored="44" name="GRP-Smoke" tests="14" failures="0" timestamp="24 Jun 2020 07:54:23 GMT" time="0.109" errors="0">
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;env&amp;apos; is required by BeforeTest on method Report but has not been marked @Optional or defined
in C:\Users\Lenovo\AppData\Local\Temp\testng-eclipse-597010918\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'env' is required by BeforeTest on method Report but has not been marked @Optional or defined
in C:\Users\Lenovo\AppData\Local\Temp\testng-eclipse-597010918\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:290)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:359)
at org.testng.internal.Parameters.createParameters(Parameters.java:620)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:190)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:209)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;env&amp;apos; is required by BeforeTest on method init but has not been marked @Optional or defined
in C:\Users\Lenovo\AppData\Local\Temp\testng-eclipse-597010918\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'env' is required by BeforeTest on method init but has not been marked @Optional or defined
in C:\Users\Lenovo\AppData\Local\Temp\testng-eclipse-597010918\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:290)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:359)
at org.testng.internal.Parameters.createParameters(Parameters.java:620)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:190)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:209)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@AfterMethod getResult" time="0.016" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CloudBlaze.Base.BaseClass.getResult(BaseClass.java:215)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:533)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="verifySourceMySql" time="0.0" classname="com.Fullload_CDC.MysqlToAdlsDM">
    <skipped/>
  </testcase> <!-- verifySourceMySql -->
  <testcase name="VerifyUserRolesPage" time="0.0" classname="com.CloudBlaze.UserSecurity.UserRoles">
    <skipped/>
  </testcase> <!-- VerifyUserRolesPage -->
  <testcase name="verifySourceAdd" time="0.0" classname="com.Fullload_CDC.OracleToAdlsDM">
    <skipped/>
  </testcase> <!-- verifySourceAdd -->
  <testcase name="verifyHomePageLink" time="0.0" classname="com.CloudBlaze.Onboarding.Home">
    <skipped/>
  </testcase> <!-- verifyHomePageLink -->
  <testcase name="verifySourceFile" time="0.0" classname="com.Fullload_CDC.FiletoAdls">
    <skipped/>
  </testcase> <!-- verifySourceFile -->
  <testcase name="verifySourceAdd" time="0.0" classname="com.Fullload_CDC.OracletoAzureSql">
    <skipped/>
  </testcase> <!-- verifySourceAdd -->
  <testcase name="verifySourceAdd" time="0.0" classname="com.Fullload_CDC.ChangeDataCapture">
    <skipped/>
  </testcase> <!-- verifySourceAdd -->
  <testcase name="verifySourceAdd" time="0.0" classname="com.Fullload_CDC.SqltoAzuresql">
    <skipped/>
  </testcase> <!-- verifySourceAdd -->
  <testcase name="verifyDataSourcesPageLink" time="0.0" classname="com.CloudBlaze.Onboarding.DataSources">
    <skipped/>
  </testcase> <!-- verifyDataSourcesPageLink -->
  <testcase name="verifySourceMySql" time="0.0" classname="com.Fullload_CDC.MySqlToAdls">
    <skipped/>
  </testcase> <!-- verifySourceMySql -->
  <testcase name="verifySourceMySql" time="0.0" classname="com.Fullload_CDC.MysqlToAdlsCDC">
    <skipped/>
  </testcase> <!-- verifySourceMySql -->
  <testcase name="verifyCustomOperationPage" time="0.0" classname="com.CloudBlaze.Onboarding.CustomOperation">
    <skipped/>
  </testcase> <!-- verifyCustomOperationPage -->
  <testcase name="verifySourceAdd" time="0.0" classname="com.Fullload_CDC.OracleToAdls">
    <skipped/>
  </testcase> <!-- verifySourceAdd -->
  <testcase name="verifySourceMySql" time="0.0" classname="com.Fullload_CDC.MySqlToSql">
    <skipped/>
  </testcase> <!-- verifySourceMySql -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest Report" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest Report -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
  <testcase name="@BeforeTest init" time="0.0" classname="com.CloudBlaze.Base.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest init -->
</testsuite> <!-- GRP-Smoke -->
